internal/functions/new/new.go

```diff
@@ -39,6 +43,23 @@ func Run(ctx context.Context, slug string, fsys afero.Fs) error {
 		}
 	}
 
+	// 3. Set up VS Code Settings
+	{
+		vscodeDir := utils.FunctionsDir + "/.vscode"
+		if _, err := fsys.Stat(vscodeDir); !errors.Is(err, os.ErrNotExist) {
+			return nil
+		}
+		if err := utils.MkdirIfNotExistFS(fsys, vscodeDir); err != nil {
+			return err
+		}
+		if err := afero.WriteFile(fsys, filepath.Join(vscodeDir, "extensions.json"), []byte(vscodeExtensions), 0644); err != nil {
+			return err
+		}
+		if err := afero.WriteFile(fsys, filepath.Join(vscodeDir, "settings.json"), []byte(vscodeSettings), 0644); err != nil {
```

Comment:
Currently this line is only reached if no `.vscode` directory exists at this location (see line 49 above), so overwriting files shouldn't be possible.

Agreed that appending the settings would be ideal, but it's a bit more complicated than the gitignore since the content is json. 

but since this is only the settings for the functions folder, I think we can start with this as an initial low hanging fruit, wdyt?